### PR TITLE
Added cloud formation template

### PR DESCRIPTION
**Pull request by ErlanBazarov**
Added AWS cloud formation template

### CODE REVIEW

File: cloudFormationTemplate.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            npm install -g grunt-cli
+            npm install -g bower
+            npm install || true
+            bower install --config.interactive=false --allow-root || true
+            echo $'[mongodb-org-4.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/amazon/2/mongodb-org/4.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc' > /etc/yum.repos.d/mongodb-org-4.4.repo
+            yum install -y mongodb-org
+            systemctl start mongod
+            cd config
+            echo "module.exports.port = 80" >> config.js
+            cd ..
```
Comment (stephenhuh) [inline cloudFormationTemplate.yaml]: generally i think we should not use relative paths like this

File: cloudFormationTemplate.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            cd sysadmins.io
+            npm install -g grunt-cli
+            npm install -g bower
+            npm install || true
+            bower install --config.interactive=false --allow-root || true
+            echo $'[mongodb-org-4.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/amazon/2/mongodb-org/4.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc' > /etc/yum.repos.d/mongodb-org-4.4.repo
+            yum install -y mongodb-org
+            systemctl start mongod
+            cd config
+            echo "module.exports.port = 80" >> config.js
```
Comment (stephenhuh) [inline cloudFormationTemplate.yaml]: hmm do we need this -- any way we can just edit the config.js file? seems like a weird thing that doesnt reallly belong in this source code.

**Review by stephenhuh — CHANGES_REQUESTED** [review]

File: cloudFormationTemplate.yaml (hunk 2 above)
Comment (ErlanBazarov) [reply cloudFormationTemplate.yaml]: I could modify the config.js file to have port 80 when the environment is production.

**Comment by stephenhuh** [comment]
Yes that is the way to go if you want to do it here. Port forwarding also
works via a proxy or some other service

File: cloudFormationTemplate.yaml (hunk 1 above)
Comment (ErlanBazarov) [reply cloudFormationTemplate.yaml]: If I edit the config.js file prior to deployment, I won't need these three lines anymore. However, what about 'cd sysadmins.io'? How can I execute 'npm install' and 'bower install' without being in the project folder?

**Comment by stephenhuh** [comment]
that particular one makes sense to me and not worth the complication -- though
you could still make it an absolute path i guess, that way the lines in the bash
script can be moed  around with no problems.
On Sat, Oct 3, 2020 at 12:56 PM ErlanBazarov <notifications@github.com> wrote:


@ErlanBazarov  commented on this pull request.


--------------------------------------------------------------------------------

In cloudFormationTemplate.yaml:

> +            curl -L https://git.io/n-install | bash -s -- -y
+            # Reload Path and environmental variables
+            source ~/.bashrc 
+            n 0.10.28
+            git clone https://github.com/stevie-sh/sysadmins.io.git
+            cd sysadmins.io
+            npm install -g grunt-cli
+            npm install -g bower
+            npm install || true
+            bower install --config.interactive=false --allow-root || true
+            echo $'[mongodb-org-4.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/amazon/2/mongodb-org/4.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc' > /etc/yum.repos.d/mongodb-org-4.4.repo
+            yum install -y mongodb-org
+            systemctl start mongod
+            cd config
+            echo "module.exports.port = 80" >> config.js
+            cd ..


If I edit the config.js file prior to deployment, I won't need these three lines
anymore. However, what about 'cd sysadmins.io'? How can I execute 'npm install'
and 'bower install' without being in the project folder?

File: cloudFormationTemplate.yaml (hunk 2 above)
Comment (stephenhuh) [reply cloudFormationTemplate.yaml]: yep!

**Pull request merged**